CMakeLists.txt

```diff
@@ -280,6 +280,7 @@ if(ENABLE_TRACY)
     include(tracy)
 endif()
 
+add_subdirectory(tt_fabric)
```

Comment:
I think it makes more sense to move this under tt_metal

Reply:
We wanted to make a separate directory, because there will be a lot more components coming (i.e. simulators, visualizers) that don't belong in tt_metal.